.github/TEMPLATE_SYNC_PAT.md

```
@@ -0,0 +1,56 @@
+              token: ${{ secrets.<secret_name> }}
+
+          - name: actions-template-sync
+            uses: AndreasAugustin/actions-template-sync@v2
+            with:
+              github_token: ${{ secrets.GITHUB_TOKEN }}
+              source_repo_path: <owner/repo>
+              upstream_branch: <target_branch> # defaults to main
+              pr_labels: <label1>,<label2>[,...] # optional, no default
+    ```
+
+4. If you encounter the error `pull request create failed: Actions is not permitted to create or approve pull requests (createPullRequest)`, follow these additional steps:
```

Markdownlint

Expected: 3; Actual: 4; Style: 1/2/3
Ordered list item prefix

(MD029, ol-prefix)